### DISCUSSION TITLE
Where to put static images in Quarto books

### DISCUSSION THREAD

**Discussion by harrelfe** [quarto-dev/quarto-cli#1168] [Q&A] [answered]
I have some small images such as icons in the right margin that hyperlink to external web pages, linked with `<img src=...>`.  I am used to putting these `.png` images in the same directory as the document but putting them inside `_book` will result in their being deleted by Quarto upon re-rendering of the book.    What is the best place to put these little `png` files such that when `_book` is on a web server the files will be found?  Ideally the images would also preview correctly when viewed locally.

Ideally the images would be stored inside the rendered `.html` files or would not create any internet loading overhead.

**Comment by jjallaire** [comment, marked as answer]
You should just put them alongside the document that use them. If they are referenced from within output (e.g. `<img src=...>` as you indicated) then they will be discovered and copied to `_book`). If any files are for whatever reason **not** discovered you can force them to be copied to `_book` you can use the `resources` option (either at the project or document level) to make sure they are copied (https://quarto.org/docs/websites/website-tools.html#site-resources)

**Reply by harrelfe** [reply]
Thanks JJ - I should have tried this simplest approach first.
